docs/designs/scrapping-workflow-phase-1-design.md

```diff
@@ -0,0 +1,67 @@
+## Scrapping workflow phase 1 design 
+
+**General requirements**
+* Workflow can support multiple sources/multiple scraping strategies
+* Workflow can be scaled easily
+* Workflow is observable
+
+**What currently exists**
+* From [this Design Doc](https://www.notion.so/de8ece87473b4f37a0ea037d607d910a?v=1bec41f87a6b42acb639d348d2282195&p=6d6b1797931e465d9cf6f5d4641c911e), Angostura can be used to publish and persist data to google BigQuery.
+* Sambil has been packaging their code as a library: c4v-py (this repo), which can be imported and used elsewhere.
```

Comment:
@Edilmo  I also tried searching for it on pip and had no luck..

Reply:
if you don't currently publish anything on pip, we can start packaging and publishing as part of building the mvp.

Reply:
@VKorelsky That's the plan!